### PR TITLE
fix: transfers with .call excluded from warning as low level code

### PR DESCRIPTION
**Pull request by dbale-altoros**
This PR comes from 
https://github.com/protofire/solhint/pull/329
All credits to @potomak

### CODE REVIEW

**Comment by potomak** [comment]
@dbale-altoros thanks for the credit.
If you want I can author another commit.

**Review by pfmescher — APPROVED** [review]

File: docs/rules/security/avoid-low-level-calls.md [hunk 1]
```diff
@@ -28,12 +28,21 @@ This rule accepts a string option of rule severity. Must be one of "error", "war
 
 
 ## Examples
+### 👍 Examples of **correct** code for this rule
+
+#### Using low level calls to transfer funds
+
+```solidity
+anyAddress.call{value: 1 ether}("");
+anyAddress.call.value(code)();
```
Comment (fvictorio) [inline docs/rules/security/avoid-low-level-calls.md:37]: `a.call.value` is deprecated.

```suggestion
anyAddress.call{value: 1 ether}("");
```

Comment (fvictorio) [reply docs/rules/security/avoid-low-level-calls.md:37]: Notice that this means updating the implementation and the tests.

Comment (dbale-altoros) [reply docs/rules/security/avoid-low-level-calls.md]: corrected

**Review by fvictorio — APPROVED** [review]

**Pull request merged**